src/components/CreatePage/PlayerPreview.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+PlayerPreview.propTypes = {
+  creatureEntry: PropTypes.object.isRequired,
+  handleDelete: PropTypes.func.isRequired,
+  handleEdit: PropTypes.func.isRequired,
+}
+
+export default function PlayerPreview({
+  handleDelete,
+  handleEdit,
+  creatureEntry,
+}) {
+  return (
+    <div>
```

Comment:
If the `div` is solely used to group these together, you might want to use a fragment instead as it does not show up in the final build